### PR TITLE
UPDATE antora-playbook.yaml and package.json

### PR DESCRIPTION
**Pull request by robnester-rh**
Update antora-playbook.yaml and package.json to use new tekton-task-antora-extension.

### CODE REVIEW

File: antora-playbook.yml [hunk 1]
```diff
@@ -29,7 +27,7 @@ content:
     - url: .
       start_path: docs
       branches: HEAD
-    - url: https://github.com/hacbs-contract/ec-policies.git
+    - url: ../ec-policies
```
Comment (simonbaird) [inline antora-playbook.yml]: Bogus diff here iiuc.

**Review by simonbaird — CHANGES_REQUESTED** [review]

File: antora-playbook.yml (hunk 1 above)
Comment (robnester-rh) [reply antora-playbook.yml]: ack. I'll update

File: package-lock.json [hunk 2]
```diff
@@ -2489,1951 +2496,5 @@
...
-    },
-    "yazl": {
-      "version": "2.5.1",
-      "resolved": "https://registry.npmjs.org/yazl/-/yazl-2.5.1.tgz",
-      "integrity": "sha512-phENi2PLiHnHb6QBVot+dJnaAZ0xosj7p3fWl+znIjBDlnMI2PsZCJZ306BPTFOaHf5qdDEI8x5qFrSOBN5vrw==",
-      "dev": true,
-      "requires": {
-        "buffer-crc32": "~0.2.3"
-      }
-    }
```
Comment (simonbaird) [inline package-lock.json:4437]: Is this expected??

**Comment by simonbaird** [comment]
Changes are related to moving to package log version 3, and it seems to still work without all the antora deps being listed.

**Review by simonbaird — APPROVED** [review]

**Comment by simonbaird** [comment]
This worked for me locally, so maybe it's fine.

```
$ rm -rf node_modules && npm ci && npm run build
...
Site generation complete!
```

**Pull request merged**